src/compiler/CombineJavascriptsCompiler.ts

```diff
@@ -0,0 +1,160 @@
+import * as path from 'path';
+
+import { CompilationResult, ICompiler } from './interfaces';
+import { cerr, debug, formatDuration, GREEN_CHECK } from '../utils';
+import * as fs from 'fs';
+import rimraf = require('rimraf');
+import * as webpack from 'webpack';
+import { Configuration } from 'webpack';
+
+export class CombineJavascriptsCompiler implements ICompiler {
+    public static readonly OUTPUT_DIR = '_generated_';
+    public static readonly ENTRY_FILE_NAME = 'javaScriptIncludesToBeCompressed.txt';
```

Comment:
Why is this file still needed after we publish all the assets to jFrog? There should be only 2 JS files left...`vendors.js` and `tsc.js`. And if we want we can combine them with this compiler

Reply:
We cannot remove it yet since most of the repositories still use this file for the imports, instead of an `index.ts` file.

Reply:
Yeah, true. Do we actually want to compress everything into one single JS file? Maybe it makes more sense to split it ([Code Splitting](https://webpack.js.org/guides/code-splitting/)). The cplace load time could be faster.

Reply:
This is how cplace is working. It expects one file `compressed.js` per plugin. We could look into code splitting later, as it could turn out to be a bigger task.